### DISCUSSION TITLE
How to send messages using WebSocket

### DISCUSSION THREAD

**Discussion by Martin-K-Kamir** [reduxjs/redux-toolkit#3779] [Q&A] [answered]
Hi, I am new to rtk query and WebSocket API. I just read the streaming updates page about getting messages from WebSocket connection. I have tried the examples on my own and it works but then I was wondering how with the rtk query I could send a message to websocket. For example, making real-time chat applications like said in docs?

**Comment by Dovakeidy** [comment, marked as answer]
I gave a basic example in this discussion: #3693

**Reply by Martin-K-Kamir** [reply]
@Dovakeidy Thanks!
